classes/plugins/__init__.py

```diff
@@ -0,0 +1,49 @@
+from multiprocessing import Array
```

Comment:
I still don't like this code being in `__init__.py`. I understand your difficulty finding an alternative, so I will try to find one in a bit.